config/middleware/receive-message/map-request-params.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const configVars = {};
+
+configVars.client = 'gambit-conversations';
+configVars.containerProperty = 'body';
+configVars.emojiStripParam = 'text';
+configVars.lowercaseParam = 'text';
+configVars.paramsMap = {
+  keyword: 'keyword',
+  broadcastId: 'broadcast_id',
+  text: 'incoming_message',
+  mediaUrl: 'incoming_image_url',
```

Comment:
Passing a `campaignId` here does return a rendered reply message -- but changing the `campaignId` parameter value isn't changing the Signup / Campaign accordingly, [it still loads the Campaign from the User.current_campaign. ](https://github.com/DoSomething/gambit/blob/703fa6d59bde8db4f8ab47247c0e4e7e64ef4804/lib/middleware/user-get.js#L13)